rules/Symfony44/Rector/ClassMethod/ConsoleExecuteReturnIntRector.php

```diff
@@ -90,6 +90,8 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        $this->hasChanged = false;
```

Comment:
This was a case of shared state where the results of a previous test would affect the next tests. By adding this statement we make sure that all tests run independently